cores/esp8266/time.cpp

```diff
@@ -228,6 +241,19 @@ void configTime(const char* tz, const char* server1, const char* server2, const
     sntp_init();
 }
 
+void configTime(const char* tz, String server1, String server2, String server3)
+{
+    static String servers[3];
```

Comment:
:+1:
I assume these `static` arrays are not linked in when these `configTime()` overloads are not used.

Reply:
In this case, yes.